src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1544,6 +1560,26 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundPattern
+    {
+        protected override abstract ImmutableArray<IOperation> Children { get; }
+    }
+
+    internal partial class BoundDeclarationPattern
+    {
+        protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.VariableAccess);
```

Comment:
I guess, it is fine to expose that node until we figure out how declaration pattern is supposed to be represented.